client/src/pages/ChefProfile.js

```diff
@@ -60,16 +61,23 @@ const useStyles = makeStyles((theme) => ({
         borderBottom: "1px solid #DCDCDC",
         background: "#ffffff",
     },
+    box: {
+        color: "#ffff",
+        background: theme.background.secondary,
+    },
 }));
 
 function ChefProfile(props) {
+    const location = useLocation();
     const user = React.useContext(UserContext);
     const classes = useStyles();
-    const { chefId } = useParams();
-
-    const chef = { ...user.profile.chefProfile };
     const [meals, setMeals] = React.useState([]);
     const [mealFormOpen, setMealFormOpen] = React.useState(false);
+    // const [chefInfo, setChefInfo] = React.useState({ ...location.state });
+    const chefInfo = location.state ? location.state : user.profile;
+    const currentChef = { ...user.profile.chefProfile };
+    const headerImage = { ...meals[0] };
```

Comment:
Will this cause problems if there are no meals yet?

Reply:
No. If there is nothing there it goes to a solid orange box